ehitchhiking-ui/src/app/shared/components/menu/menu.component.html

```diff
@@ -1,35 +1,46 @@
 <mat-sidenav-container class="sidenav-container" [ngClass]="{'sidenav-container-opened': opened}">
 	<mat-sidenav class="menu" #sidenav mode="side" [(opened)]="opened" [ngClass]="{'menu-opened': opened}">
 		<div class="header">
-			<button mat-icon-button (click)="openProfileDialog()" class="header-button">
+			<button mat-icon-button (click)="openProfileDialog()" matTooltip="open profile" class="header-button">
 				<mat-icon class="header-button-icon account-icon">account_circle</mat-icon>
 			</button>
 			<span class="header-span truncated" [matTooltip]="currentUser ? currentUser.getFullName() : 'Anonymous'">{{
 				currentUser ? currentUser.getFullName() : 'Anonymous'
 			}}</span>
 			<div class="modal-close-control">
-				<button mat-icon-button class="modal-close-control-btn header-button" (click)="sidenav.toggle()">
+				<button
+					mat-icon-button
+					class="modal-close-control-btn header-button"
+					matTooltip="close"
+					(click)="sidenav.toggle()"
+				>
 					<mat-icon class="header-button-icon close-icon">close</mat-icon>
 				</button>
 			</div>
 		</div>
 		<div class="buttons">
 			<li class="li-buttons">
-				<button mat-raised-button class="li-button" (click)="openHistoryDialog()">
+				<button mat-raised-button matTooltip="open history" class="li-button" (click)="openHistoryDialog()">
```

Comment:
this button has a text that absolutely equals to tooltip message. Tooltip isn't required here